src/rest-server/src/templates/dockerContainerScript.mustache

```diff
@@ -29,10 +29,17 @@ function exit_handler()
     "[DEBUG]" "Docker container exit handler: EXIT signal received in docker container, exiting ..."
 }
 
+function kill_handler()
+{
+  printf "%s %s\n" \
+    "[DEBUG]" "Docker container kill handler: SIGTERM signal received in docker container, exiting ..."
+    exit 0
```

Comment:
Don't `exit 0`, it will be considered as a succeeded job. 
Also, you could dump some log like "consume too much space" and suggest to use hdfs or nfs.

Reply:
Accepted